MODULE.bazel

```diff
@@ -30,6 +30,16 @@ register_toolchains(
     "@rules_perl//perl:perl_windows_x86_64_toolchain",
 )
 
+cpan = use_extension("@rules_perl//perl/cpan:extensions.bzl", "perl_cpan")
```

Comment:
I think `cpan` would be preferable to `perl_cpan` to reduce the smurfing.
Compare also with `crate = use_extension("@rules_rust//crate_universe:extension.bzl", "crate")` and `pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")`

Reply:
Updated!